api/gateway.proto

```diff
@@ -239,4 +240,12 @@ message GatewayConnectionStats {
   uint64 uplink_count = 6;
   google.protobuf.Timestamp last_downlink_received_at = 7 [(gogoproto.stdtime) = true];
   uint64 downlink_count = 8;
+
+  message RoundTripTimes {
+    google.protobuf.Duration min = 1 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true, (validate.rules).message.required = true];
+    google.protobuf.Duration max = 2 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true, (validate.rules).message.required = true];
+    google.protobuf.Duration average = 3 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true, (validate.rules).message.required = true];
```

Comment:
IMO a `median` is more useful than an `average` (which could be biased because of a single high measurement).

Reply:
It is the median. It's not the arithmetic mean. Both are considered "average"

Reply:
I agree with @johanstokking, documenting that it's median wouldn't hurt though

Reply:
I'll make it `median` explicitly